packages/pattern-picker/package.json

```diff
@@ -33,6 +33,7 @@
 		"@automattic/onboarding": "workspace:^",
 		"classnames": "^2.3.1",
 		"react-query": "^3.32.1",
+		"swiper": "^8.4.5",
```

Comment:
This package (and design-carosel) were relying on the fact that `swiper` was in the dependency tree, without actually depending on them.